scipy/sparse/dia.py

```diff
@@ -237,6 +237,40 @@ def sum(self, axis=None, dtype=None, out=None):
 
     sum.__doc__ = spmatrix.sum.__doc__
 
+    def _add_sparse(self, other):
+        """If other is also of type dia_matrix,
```

Comment:
A docstring isn't strictly necessary for internal methods like this one, and I think the inline comments do a better job of describing what is being done here, so I'd prefer to leave this out. The rationale is that comments close to the code they're describing are more likely to be updated when things change in the future.

Reply:
ok